src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -826,8 +826,33 @@ using namespace ngraph::runtime;
                     }
                     return false;
                 };
-                // Always enable nodes computing output tensors
-                if (computes_output())
+
+                auto possibly_overwritten = [&]() {
+                    for (const descriptor::Output& output : node->get_outputs())
+                    {
+                        for (const descriptor::Input* input : output.get_inputs())
```

Comment:
At this point, we should be able to confidently tell if an input was overwritten or not. Do we need to be conservative here?